llvmlite/tests/test_binding.py

```diff
@@ -2701,6 +2701,14 @@ def test_literal_struct(self):
             # Also test constant text repr
             gv.initializer = ir.Constant(typ, [1])
 
+        # Packed layout2
```

Comment:
I think this is the intention of this added part of the test:
```suggestion
        # Packed layout created from Constant.literal_struct
```